bookshelter-app/src/pages/Home/index.tsx

```diff
@@ -4,22 +4,43 @@ import { get } from '../../services/api';
 import { useState, useEffect } from 'react';
 import { Book } from '../../types/index';
 import { API } from '../../constants/api';
+import { useParams, useNavigate } from 'react-router-dom';
+
 const HomePage = () => {
   const [books, setBooks] = useState<Book[]>([]);
+  const { slug, name } = useParams<{ slug: string; name: string }>();
+  const navigate = useNavigate();
 
   useEffect(() => {
     const fetchBookList = async (): Promise<void> => {
       const listBooks = await get<Book[]>(API.BOOKS_ENDPOINT);
```

Comment:
use https://github.com/mockapi-io/docs/wiki/Quick-start-guide#filtering to get book by category

Reply:
fixed!